Libraries/LibGUI/ColorButton.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2020-2020, Hüseyin Aslıtürk <asliturk@hotmail.com>
```

Comment:
A simple `2020` would suffice here in copyright headers instead of this `2020-2020` year range.